templates/Primitives+ExtensionFile.stencil

```diff
@@ -16,3 +16,12 @@ extension Int64: LocalizedError {
     public var errorDescription: String? { return String(self) }
 }
 
+extension Data {
+    func base64URLEncodedString() -> String {
+        let base64String = self.base64EncodedString()
+        let base64UrlString = base64String.replacingOccurrences(of: "/", with: "_")
+                        .replacingOccurrences(of: "+", with: "-")
+                        .replacingOccurrences(of: "=", with: "")
+        return base64UrlString
+    }
+}
```

Comment:
Is this a workaround for the `+` issue? Should this be in AzureCore?

Reply:
Yes, this should be in AzureCore.